herd/AArch64Arch_herd.ml

```diff
@@ -145,6 +145,9 @@ module Make (C:Arch_herd.Config) (V:Value.S) =
       | I_TLBI (_,_)
           -> None
 
+    let default_memattr = V.memattrToV [ "Normal"; "Inner-write-back"; "Outer-write-back";
```

Comment:
Is this default list of attributes the one of kvm-unit-tests ?

Reply:
That's what you get when you do a malloc from user space. I thought this is a reasonable default that also maintains backwards compatibility.